pclntab/pclntab_test.go

```diff
@@ -52,7 +53,7 @@ func TestGoPCLnTabExtraction(t *testing.T) {
 			t.Run(fmt.Sprintf("go1.%v#%v", goMinorVersion, name), func(t *testing.T) {
 				t.Parallel()
 				exe := filepath.Join(tmpDir, fmt.Sprintf("%v.v1_%v.%v", strings.TrimRight(srcFile, ".go"), goMinorVersion, name))
-				cmd := exec.Command("go", append([]string{"build", "-o", exe}, test.buildArgs...)...) // #nosec G204
+				cmd := exec.CommandContext(context.TODO(), "go", append([]string{"build", "-o", exe}, test.buildArgs...)...) // #nosec G204
```

Comment:
In test you should be able to pass the test context: https://pkg.go.dev/testing#T.Context